ext/pcntl/tests/pcntl_exec_restrict.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+pcntl_exec() - Testing with open_basedir restriction.
+--EXTENSIONS--
+pcntl
+--INI--
+open_basedir=${HOME}/bin
+--SKIPIF--
+<?php
+if (!getenv("TEST_PHP_EXECUTABLE") || !is_executable(getenv("TEST_PHP_EXECUTABLE"))) die("skip TEST_PHP_EXECUTABLE not set");
```

Comment:
Although not likely,  `$TEST_PHP_EXECUTABLE` might be under `$HOME/bin`.

Perhaps ensure that test will always be correct regardless of environment it runs in.